client/README.md

```diff
@@ -0,0 +1,8 @@
+To install and run
+
+```
+yarn
+yarn start
+```
+
+We don't have lint-staged, or any other method to enforce styling as a pre-commit hook (lint-staged does not work well without it being setup in root dir), so you might want to setup prettier/eslint in your editor of choice (or run the fix script before submitting PR).
```

Comment:
Please add section about css (to use the same name as you use for scss).
Also a section showing that we use jest is nice (because it is hidden in create react app).